### PR TITLE
Adding support windows stemcells

### PR DESCRIPTION
**Pull request by bbucher-sigsci**
https://github.com/signalsciences/sigsci/issues/23830

Ports the existing buildpack setup scripts to powershell in order to support Windows in Tanzu.

### CODE REVIEW

File: bin/.profile.d/sigsci-agent.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@echo off
+powershell.exe -ExecutionPolicy Unrestricted -File "/Users/bbucher-sigsci/work/sigsci-cloudfoundry-buildpack/bin/.profile.d/sigsci-agent.ps1"
```
Comment (leaf-fastly) [inline bin/.profile.d/sigsci-agent.bat]: Should this be 
`powershell.exe -ExecutionPolicy Unrestricted -File "%~dp0\sigsci-agent.ps1"`

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ./sigsci-cloudfoundry-buildpack.tar.gz
-          asset_name: sigsci-cloudfoundry-buildpack.tar.gz
-          asset_content_type: application/gzip
+          asset_path: ./sigsci-cloudfoundry-buildpack.zip
+          asset_name: sigsci-cloudfoundry-buildpack.zip
+          asset_content_type: application/zip
```
Comment (leaf-fastly) [inline .github/workflows/release.yml]: Is there a reason to not produce .zip and .tar.gz?

**Review by leaf-fastly — CHANGES_REQUESTED** [review]
I didn't see the reasoning for switching from .tar.gz to .zip, happy to be talked into it, or have you add back .tar.gz alongside .zip

File: .github/workflows/release.yml (hunk 2 above)
Comment (bbucher-sigsci) [reply .github/workflows/release.yml]: None. I think I was getting confused from working in pwsh for so long. They only support zip. Changed these back to use gzip.

**Review by leaf-fastly — APPROVED** [review]

**Pull request merged**